src/main/java/TaskManager.java

```diff
@@ -0,0 +1,50 @@
+public class TaskManager {
+    private static int taskNo = 0;
+    final private static int maxTasks = 100;
+    final static Task[] tasks = new Task[maxTasks];
```

Comment:
Perhaps this should be `private`, following the [encapsulation paradigm of OOP](https://nus-cs2113-ay2122s1.github.io/website/schedule/week3/topics.html#paradigms-oop-objects-encapsulation-of-objects)?
```suggestion
    private static final Task[] tasks = new Task[MAX_TASKS];
```